DOM_creation/Lesson2/main.js

```diff
@@ -0,0 +1,15 @@
+const ul= document.getElementById("js-ul");
+const li = document.createElement('li');
+
+const a = document.createElement('a');
+a.setAttribute("href", "1.html");
+const text = document.createTextNode("これです");
+
+const img = document.createElement('img');
+img.setAttribute("src", "bookmark.png");
+img.setAttribute("alt", "ブックマーク");
+
+a.appendChild(img);
+a.appendChild(text);
+li.appendChild(a);
+ul.appendChild(li);
```

Comment:
IMO
How about connecting here?

```suggestion
ul.appendChild(li).appendChild(a).appendChild(img).appendChild(text);
```

Reply:
Thank you Chihiro-san,

I chained them in one code, but I am facing an issue. When chaining all codes together, the Text: これです was included within img tags(opening and closure). And when it was rendered in HTML, the text was not displayed. Is there any ways or method to create text after img tag without an img closure tag?